### ISSUE TITLE
LinkedIn OAuth Internal Server Error

### ISSUE THREAD

**Issue by mahdiendigitals** [nhost/hasura-backend-plus#649]
![image](https://user-images.githubusercontent.com/90712531/134136864-32a73c85-2361-4d16-bdbf-79ec6e960923.png)

**Comment by elitan** [comment]
the URL in the browser address bar looks a bit strange: `[..]/callbackk` should be `[..]/callback`.

**Comment by mahdiendigitals** [comment]
yes sorry it's correct in the code but when i took the screen shot made a mistake still the same problem

**Comment by elitan** [comment]
Can you send me an email with the full backend URL so I can check the logs and debug this?

johan@nhost.io
